soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
+     Text{"Progressive Darknut Sword", "Épée Darknut (prog.)", "Espada Darknut progresiva"},
+     Text{"Progressive Power Sword", "Épée de puissance (prog.)", "Espada de poder progresiva"},
+     Text{"Progressive Big Stabby", "Gros coup de poignard (prog.)", "Gran puñalada progresiva"}};
 
   trickNameTable[0x0F] = {
-     Text{"Magic Bottle", "Flacon magique", "Frasco feérico"},
-     Text{"Glass Bottle", "Flacon de verre", "Botella de cristal"},
-     Text{"Bottle with Water", "Flacon d'eau", "Botella Tingle"}};
+     Text{"Empty Canteen", "Cantine vide", "cantimplora vacía"},
+     Text{"Vial of Winds", "Fiole de vents", "Vial de Vientos"},
+     Text{"Tingle Bottle", "Flacon de Tingle", "Botella de Tingle"}};
   trickNameTable[0x14] = {
      Text{"Bottle with Chateau Romani", "Flacon de cuvée Romani", "Botella de Reserva Romani"},
-     Text{"Bottle with Fresh Milk", "Flacon de lait frais", "Botella de leche fresca"},
+     Text{"Bottle with Premium Milk", "Flacon avec lait de qualité supérieure", "Biberón con leche Premium"},
      Text{"Bottle with Mystery Milk", "Flacon de lait grand cru", "Botella de leche extra"}};
   trickNameTable[0x8C] = {
      Text{"Bottle with Red Chu Jelly", "Flacon de gelée Chuchu rouge", "Jugo de Chuchu Rojo"},
-     Text{"Bottle with Medicine of Life", "Flacon d'élixir rouge", "Botella de medicina de la vida"},
+     Text{"Bottle with Hibiscus Potion", "Flacon de potion de Hibiscus", "Botella de poción de Hibisco"},
      Text{"Bottle with Heart Potion", "Flacon de potion de soin", "Botella de poción de salud"}};
   trickNameTable[0x8D] = {
      Text{"Bottle with Green Chu Jelly", "Flacon de gelée Chuchu verte", "Jugo de Chuchu Verde"},
-     Text{"Bottle with Medicine of Magic", "Flacon d'élixir vert", "Botella de medicina mágica"},
+     Text{"Bottle with Lamp Oil", "Flacon de Huile à lanterne", "Botella de Aceite de candil "},
      Text{"Bottle with Stamina Potion", "Flacon d'Endurol", "Botella de elixir vigorizante"}};
   trickNameTable[0x8E] = {
      Text{"Bottle with Blue Chu Jelly", "Flacon de gelée Chuchu bleue", "Jugo de Chuchu Azul"},
      Text{"Bottle with Water of Life", "Flacon d'élixir bleu", "Botella de agua de la vida"},
      Text{"Bottle with Air Potion", "Flacon de potion d'oxygène", "Botella de oxígeno"}};
   trickNameTable[0x8F] = {
      Text{"Bottle with Forest Firefly", "Flacon avec une luciole", "Luciérnaga del bosque"},
-     Text{"Bottle with Faerie", "Flacon de poudre féérique", "Gran Hada embotellada"},
+     Text{"Bottle with Deku Princess", "Flacon avec Deku Princess", "Botella con Deku Princess"},
      Text{"Bottle with Stray Fairy", "Flacon avec une fée perdue", "Hada perdida en una botella"}};
   trickNameTable[0x90] = {
-     Text{"Bottle with Small Jabu-Jabu", "Flacon avec mini Jabu-Jabu", "Lord Chapu-Chapu embotellado"},
+     Text{"Bottle with Reekfish", "Flacon avec Reekfish", "Reekfish embotellada"},
```

Comment:
I like the TP ref;; but the original one was better about an obscure cut content about MM and a mini jabu jabu on a bottle

Reply:
Did not know about that reference, thought it was just a joke, my bad